flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/systems.cfg

```diff
@@ -73,32 +73,19 @@ lightdef.19 = Type:10#Index:2#LocalPosition:34.8,1.0,3.7#LocalRotation:270,500,0
 ; Pedestal flood light
 lightdef.20 = Type:11#Index:1#LocalPosition:35.2,0,8.5#LocalRotation:80,0,0#EffectFile:LIGHT_A32NX_Pedestal#PotentiometerIndex:76
 
-; Captain left glareshield light
-lightdef.21 = Type:12#Index:3#LocalPosition:38.97,-2.550,5.87#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
-
-; Captain right glareshield light
-lightdef.22 = Type:12#Index:3#LocalPosition:38.97,-0.6,6.2#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
-
-; F/O left glareshield light
-lightdef.23 = Type:12#Index:4#LocalPosition:38.97,0.7,6.2#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
-
-; F/O right glareshield light
-lightdef.24 = Type:12#Index:4#LocalPosition:38.97,2.55,5.87#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
-
-; Overall glareshield (fake) ambient light
-lightdef.25 = Type:12#Index:3#LocalPosition:36.5,0,6.5#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiant#PotentiometerIndex:83
-
-; Captain left glareshield (fake) ambient light
-lightdef.26 = Type:12#Index:3#LocalPosition:37.97,-2.45,4.87#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiantLow#PotentiometerIndex:83
-
-; Captain right glareshield (fake) ambient light
-lightdef.27 = Type:12#Index:3#LocalPosition:38,-0.6,4.8#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiantLow#PotentiometerIndex:83
-
-; F/O left glareshield (fake) ambient light
-lightdef.28 = Type:12#Index:4#LocalPosition:38,0.7,4.8#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiantLow#PotentiometerIndex:83
-
-; F/O right glareshield (fake) ambient light
-lightdef.29 = Type:12#Index:4#LocalPosition:37.97,2.45,4.87#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiantLow#PotentiometerIndex:83
```

Comment:
**issue**: Unused effect file still in repository.
**suggestion**: Remove `LIGHT_A32NX_CockpitMinimalAmbiantLow.fx`.

Reply:
File removed.